tests/Functional/Connection/SSLConnectionTest.php

```diff
@@ -78,6 +78,12 @@ public function secure_connection_params()
             $options
         ];
 
+        // #5 TLS crypto method
```

Comment:
Need more tests with different network protocols. Some use `amqps` in example.

Reply:
Not sure what you mean. Here we're talking about the transport layer protocols (tcp/tcp+tls).
amqp/amqps is more about the application layer in terms of OSI.
In other words: there is no way to pass amqp/amqps into AMQPSSLConnection/AMQPStreamConnection, we will just get an error if amqp/amqps value is passed to the stream_socket_client function.
https://www.php.net/manual/en/transports.inet.php

Reply:
If amqps is not possible, then please proove it with tests. We had several request from community regarding those "wrappers" in the past.
Also, do we have test with keep alive enabled?

Reply:
I don't know how to prove it here, since there is no longer any way to pass amqps or any other string value, but I created a separate branch from master to show it: https://github.com/egorgrushko/php-amqplib/tree/amqp_amqps_tests

And yes, I missed the tests with keepalive enabled. Added some.

Reply:
Let's merge that test.

Reply:
That test will not work in this branch.
> there is no longer any way to pass amqps or any other string value